content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,209 @@
+
+ ID                                      | Value\[x] Type      | Description
+-----------------------------------------|---------------------|--------------------------------------------------------------------------
+ `financial-transaction-account-number`  | [`string`]          | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+ `financial-transaction-alias`           | [`string`]          | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-allocation`      | None (contains nested extensions)      | Defines how the payment or adjustment is to be allocated across other resources.
+ `financial-transaction-amount`          | [`Money`]           | The total amount of the financial transaction.
+ `financial-transaction-card-brand`      | [`string`]          | Identifies the brand of credit card when credit card is used as a payment method.
+ `financial-transaction-date`            | [`date`]            | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-location`        | [`string`]          | Client configured value representing the location or workflow that the payment was received in.
+ `financial-transaction-method`          | [`string`]          | Describes the method of payment for the financial transaction.
+ `financial-transaction-tendered-amount` | [`Money`]           | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+ `financial-transaction-type`            | [`CodeableConcept`] | The classification of the transaction.
+
+## Create
+
+Create a FinancialTransaction.
+
+    POST /Basic    
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported.
+* When integrating your application with a client's production environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction.
+* The financial-transaction-allocation.target field must reference an Encounter associated with an Account.
+* Clients must use Cerner's Revenue Cycle Patient Accounting application in order to create Financial Transactions.
+* A FinancialTransaction can only be targeted at a clinical Encounter that has related financial activity within the Cerner Revenue Cycle Patient Accounting system.
+* FinancialTransaction and Basic scopes are required
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token&gt;', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:financial_transaction, :create, :r4) %>
+
+### Example
```

Comment:
@MaxPhilips This looks a little confusing to me, there's no separation or title to these examples. Should we come up with some sort of divider for these examples?

At the very least we should probably have a title for each example so we'll be able to tell what each example is for.

Reply:
We should follow the same pattern that appointment does.

Create new headings for each Example with a small description of what the example is of: https://github.com/cerner/fhir.cerner.com/blame/master/content/millennium/r4/scheduling/appointment.md#L135

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/1da195dca470524ec6880af1b214b75d737ecabe#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR74